### PR TITLE
Comments entity for news

### PR DESCRIPTION
**Pull request by Axedez**
[Trello Task](https://trello.com/c/rtMRhAH9/83-comments-entity-for-news)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [x] screenshot of feature is added if task has UI
image
ezgif com-gif-maker

### CODE REVIEW

**Review by mmarusyk — APPROVED** [review]

File: app/javascript/components/my_osbb/News/Comments/Comments.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+p {
+  text-indent: 0;
+}
```
Comment (mmarusyk) [inline app/javascript/components/my_osbb/News/Comments/Comments.module.scss:3]: може вплинути на інші стилі. напиши клас краще

Comment (Axedez) [reply app/javascript/components/my_osbb/News/Comments/Comments.module.scss:3]: Та, я теж, коли переглядав код, про це подумав!

File: db/seeds.rb [hunk 2]
```diff
@@ -38,6 +38,14 @@
   FactoryBot.create_list(:news, 10, osbb: lead.osbb, user: lead)
 end
 
+News.all.each do |news|
+  FactoryBot.create_list(:comment, 10, commentable: news, user: User.where(osbb_id: news.osbb_id).sample)
+end
+
+News.all.each do |news|
```
Comment (chu4) [inline db/seeds.rb:45]: whats the point of going through news twice?

**Review by Ukhanskyi — APPROVED** [review]

File: app/javascript/components/my_osbb/News/Comments/EachComment.jsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+  const showMore = () => {
+    setShowSubcomments(!showSubcomments);
+  }
+
+  const handleDelete = () => {
+    deleteComment(id, parent, news_id).then(res => {
+      (res=='done') ? onChange(id) : updateParentBlock(res)
+    }).catch((err) => {
+      console.log(err);
```
Comment (chu4) [inline app/javascript/components/my_osbb/News/Comments/EachComment.jsx:48]: why do you catch errors in to console.log?

**Review by chu4 — APPROVED** [review]

**Pull request merged**